packages/mobile/src/harmony-native/components/TextLink/TextLink.tsx

```diff
@@ -45,43 +49,71 @@ export const TextLink = <ParamList extends ReactNavigation.RootParamList>(
   const variantPressingColors = {
     default: color.primary.p300,
     subdued: color.primary.p300,
-    visible: color.link.visible,
-    inverted: color.static.white,
+    visible: color.static.primary,
+    inverted: color.static.primary,
```

Comment:
were these colors incorrect before?

Reply:
They were incorrect and are still incorrect 😂  About to push the correct colors
good catch!